tests/standalone-tests/pom.xml

```diff
@@ -61,17 +61,17 @@
         <dependency>
             <groupId>org.apache.maven.resolver</groupId>
             <artifactId>maven-resolver-api</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
What's the reason for these being changed from test to provided? TBH it looks weird to me to have them at all.

Reply:
They are required for compilation, they were transitively included.

Reply:
There isn't anything that's a non-test that should need to be compiled. If test scope doesn't work it seems like something is wrong.

Reply:
@jamezp you were right, I fixed the pom.